src/app/containers/RadioSchedule/utilities/processRadioSchedule.js

```diff
@@ -20,6 +24,13 @@ export const getLink = (state, program, service) => {
   return state === 'live' ? `${url}/liveradio` : `${url}/${pid}`;
 };
 
+const logProgramError = error => {
+  logger.error({
+    event: RADIO_SCHEDULE_DATA_INCOMPLETE_ERROR,
+    message: error,
+  });
```

Comment:
```suggestion
  logger.error(RADIO_SCHEDULE_DATA_INCOMPLETE_ERROR, { error });
```
I think this format is correct. Apologies for my mistaken suggestion before!